src/ToDo/Business/Services/AuthenticationService.cs

```diff
@@ -0,0 +1,63 @@
+﻿
+using Microsoft.Identity.Client;
+using ToDo.Business.Entities;
+using Uno.UI.MSAL;
+using System.IdentityModel.Tokens.Jwt;
+
+namespace ToDo.Business.Services
+{
+	public class AuthenticationService: IAuthenticationService
+	{
+		private readonly IPublicClientApplication _pca;
+		private readonly OAuthSettings _settings;
+		public AuthenticationService(HostBuilderContext context)
+		{
+			_settings = new OAuthSettings(
+					context.Configuration["OAuthSettings:ApplicationId"],
+					context.Configuration["OAuthSettings:Scopes"],
+					context.Configuration["OAuthSettings:RedirectUri"]);
+
+			_pca = PublicClientApplicationBuilder
+					.Create(_settings.ApplicationId)
+					.WithRedirectUri(_settings.RedirectUri)
+					.WithUnoHelpers()
+#if __IOS__
+                .WithIosKeychainSecurityGroup("9TB54R6A6V.com.horus.unoplatform.todoapp")
```

Comment:
Please move all strings to constants and extract them in their own source file (can remains in this class by using a partial modifier. Exemple: `AuthenticationService.Constants.cs`.

Reply:
Could you explain about _can remains in this class by using a partial modifier. Exemple: AuthenticationService.Constants.cs._
:)